ext/BeamTrackingBeamlinesExt/exact.jl

```diff
@@ -14,4 +14,18 @@ end
 @inline function drift(tm::Exact, bunch, L)
   tilde_m, gamsqr_0, beta_0 = ExactTracking.drift_params(bunch.species, bunch.Brho_ref)
   return KernelCall(ExactTracking.exact_drift!, (beta_0, gamsqr_0, tilde_m, L))
+end
+
+@inline function thick_bend_pure_bdipole(tm::Exact, bunch, bendparams, bm1, L)
+  g = bendparams.g
+  theta = g * L
+  k0, _ = get_strengths(bm1, L, bunch.Brho_ref)
```

Comment:
```suggestion
  Kn0, Ks0 = get_strengths(bm1, L, bunch.Brho_ref)
```

And we need to be able to support nonzero `Ks0`. This probably can be done using a patch to rotate into a plane where you only have a `Kn0` and then rotate back.  If someone commits to implementing this soon then I will merge this but with add the error message:
```suggestion
  Kn0, Ks0 = get_strengths(bm1, L, bunch.Brho_ref)
  Ks0 ≈ 0 || error("Skew/tilt multipoles not implemented yet for tracking method $tm")
```

Reply:
@mattsignorelli I will implement the patching if this gets pulled in. I can also add some more tests/make it branchless if necessary.

Reply:
@JosephPeterDevlin you have write permission in this repository, so you could actually pull this branch, work on it, and then push back to this branch

Reply:
I haven't worked on this yet

Reply:
@mattsignorelli The BendParams currently only has one number for g. I assume this means the curvature in the horizontal plane, but that means we cannot currently support vertical bends, and only what you would call a vertical corrector/kicker.

Reply:
`tilt_ref` rotates the bend so you can get a vertical bend.

Reply:
What is tilt_ref? Do you mean the alignment parameter? If so, that's probably true, but that should most likely be handled when misalignments get included.

Reply:
Oh, I assume you mean ref_tilt from classic Bmad. Does that exist in SciBmad?

Reply:
Correct.

Reply:
In `SciBmad` we could have the curvature values `gx` and `gy` instead, and I guess optionally another independent variable `g_tilt`. This is all for the curvature of the coordinate system - irrespective of the bend strength.

Reply:
Yeah, the patches are not so simple then. If I'm understanding correctly, the exact bend could only be patched into if `Ks0 = 0` for any `g_tilt` and `gy = 0` or if `atan(gy/gx) = atan(Ks0/Kn0)` with `g_tilt = 0`.

Reply:
`gx` and `gy` are possible but `tilt_ref` is more flexible. I don't understand what the difference is  between `g_tilt` and `tilt_ref`

Reply:
No difference, I just think `g_tilt` is more clear than `tilt_ref` personally.

@JosephPeterDevlin For nonzero `g_tilt` one would just decompose that into solely `g_x` and `g_y`, just like we do with the multipoles in `get_strengths`. Then after that just use `g_x` and `g_y`.

Reply:
Also keep in mind that `tilt` is different from `tilt_ref` (and that no other type of element has a `tilt_ref`)

Reply:
@mattsignorelli What I mean is that the bend Alex implemented only works if `g` is in the same plane as the dipole field. A general solution for curvature in any plane might exist, but I've never seen it.

Reply:
Actually with `tilt_ref` the reference curve is being tilted. In both cases, `tilt` and `tilt_ref` will tilt `g` so `g_tilt` is a bit ambiguous.

Reply:
@JosephPeterDevlin Remember that the tracking is done in body coordinates and in body coordinates the bend rotation is always around the `y`-axis independent of `tilt_ref`. So Alex's implementation should work.

Reply:
@DavidSagan @JosephPeterDevlin can we meet on zoom to discuss?

Reply:
Sure. Go to my zoom.

Reply:
I'm available.